packages/trader/src/Assets/Trading/Categories/trade-categories.jsx

```diff
@@ -8,6 +8,46 @@ const TradeCategories = ({ category }) => {
     let TradeTypeTemplate;
     if (category) {
         switch (category) {
+            case 'accumulator':
+                TradeTypeTemplate = (
+                    <React.Fragment>
+                        <p>
```

Comment:
We need to refactor this to use <Text> component in future.
Add a TODO

Reply:
i've refactored this for accumulator and left a TODO comment for the rest👍🏻